src/renderers/WebGLRenderer.js

```diff
@@ -1937,7 +1937,7 @@ function WebGLRenderer( parameters ) {
 			//  WebGLRenderTargetCube will be flipped for backwards compatibility
 			//  WebGLRenderTargetCube.texture will be flipped because it's a Texture and NOT a CubeTexture
 			// this check must be handled differently, or removed entirely, if WebGLRenderTargetCube uses a CubeTexture in the future
-			uniforms.flipEnvMap.value = ( ! ( material.envMap && material.envMap.isCubeTexture ) ) ? 1 : - 1;
+			uniforms.flipEnvMap.value = material.envMap.isCubeTexture ? - 1 : 1;
```

Comment:
Is this change intentional?

Reply:
Yes, just to reduce the unnecessary calculations.

Reply:
What if `material.envMap` is `null`?

Reply:
Oh. True that. Thanks!